### PR TITLE
dev

### PR DESCRIPTION
**Pull request by DominiqueMakowski**

### CODE REVIEW

File: R/correlation.R [hunk 1]
```diff
@@ -36,13 +36,17 @@
...
+#' )
 #' cor <- correlation(iris)
 #'
 #' cor
 #' summary(cor)
 #' as.table(cor)
 #'
+#' # Grouped dataframe
+#' library(dplyr)
+#'
```
Comment (strengejacke) [inline R/correlation.R]: use suggested packages only condtionally...

**Comment by DominiqueMakowski** [comment]
and now it fails on mac 😠

**Comment by DominiqueMakowski** [comment]
Today is the 8th, as we said, should we submit to CRAN?

**Comment by DominiqueMakowski** [comment]
submitted

**Comment by DominiqueMakowski** [comment]
On its way to CRAN. Man that was fast!

**Comment by strengejacke** [comment]
btw, I appreciate the [news-file](https://github.com/easystats/correlation/blob/master/NEWS.md)...

**Comment by DominiqueMakowski** [comment]
It's such a piece of art that now I can't modify it 😱

**Pull request merged**